data/schema/core/actionwml.cfg

```diff
@@ -525,6 +525,18 @@
 		{INSERT_TAG}
 		{SIMPLE_KEY id string}
 		{DEFAULT_KEY remove s_bool no}
+		[if]
+			[not]
+				# As it allows s_bool, this can't test for every possible "yes" value,
+				# but if there's no remove= then it's definitely adding a tunnel.
+				glob_on_remove=*
+			[/not]
+			[then]
+				{FILTER_TAG "filter" unit min=1}
```

Comment:
c++ change says max=1

Reply:
max defaults to 1, and with this change the schema validation checks the max, but I should probably add a comment about that